x-pack/test_serverless/functional/test_suites/security/cypress/tasks/login.ts

```diff
@@ -36,8 +36,8 @@ const sendApiLoginRequest = (
         method: 'POST',
         headers,
         body: {
-          providerType: basicProvider.type,
-          providerName: basicProvider.name,
+          providerType: basicProvider!.type,
+          providerName: basicProvider!.name,
```

Comment:
Based on how this issue has been addressed [in other files](https://github.com/elastic/kibana/blob/26873aff87a62ed7c7f1681284a9e1be79d43079/x-pack/plugins/security_solution/public/management/cypress/tasks/login.ts#L102-L103), I'm pretty sure this should be like this:

```suggestion
          providerType: basicProvider?.type,
          providerName: basicProvider?.name,
```